Kickstarter-iOS/AppDelegate.swift

```diff
@@ -234,6 +234,24 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
         self?.rootTabBarController?.present(navController, animated: true)
       }
 
+    self.viewModel.outputs.verifyEmailWithURLRequest
+      .observeForUI()
+      .observeValues { [weak self] in self?.verifyEmail($0) }
+
+    self.viewModel.outputs.emailVerificationCompleted
+      .observeForUI()
+      .observeValues { [weak self] message, success in
+        // FIXME: We may want to revisit this shortcut to find the VC to present on.
```

Comment:
This isn't ideal but it was becoming quite unwieldy to do this through a whole lot of message-passing. I think we need to consider when and how this might be displayed. It's possible a notification may serve us better here 🤔

Reply:
Could we use `self.window` here? Not sure if that is better or worse that the existing implementation though.

Reply:
A similar approach to what I was originally thinking which was to use `self.window`. I think your approach is effectively the same.

Reply:
We might want to look into that for dismissing the presented VC.

Reply:
It actually works fine to display the banner on the tab bar controller so i've updated to do that.